java/src/main/java/ai/rapids/cudf/Table.java

```diff
@@ -1626,12 +1626,14 @@ public Table aggregateWindowsOverTimeRanges(WindowAggregate... windowAggregates)
      * Table t2 ...
      * Table result = t1.onColumns(0,1).leftJoin(t2.onColumns(2,3));
      * @param rightJoinIndices - Indices of the right table to join on
+     * @param compareNullsEqual - Whether null join-key values should match or not.
      * @return the joined table.  The order of the columns returned will be join columns,
      * left non-join columns, right non-join columns.
      */
-    public Table leftJoin(TableOperation rightJoinIndices) {
+    public Table leftJoin(TableOperation rightJoinIndices, boolean compareNullsEqual) {
```

Comment:
Just a nit: I am not sure that we want to ask everyone to provide the boolean in all cases.  Cudf has a default value for the parameter. I would like to see up match what cudf does for their default if possible.

Reply:
I agree with you there. This would break anyone who's already using the join methods. I wanted your opinion before I added a default.
The default would break the intention of `spark-rapids`'s joins, but that would be no worse than the current situation.

I'll add a default shortly.

Reply:
This should now be resolved.